icu4c/source/common/rbbi.cpp

```diff
@@ -1214,9 +1223,9 @@ getLanguageBreakEngineFromFactory(UChar32 c)
 //                          the character c.
 //
 //-------------------------------------------------------------------------------
-const LanguageBreakEngine *
+LanguageBreakEngine *
 RuleBasedBreakIterator::getLanguageBreakEngine(UChar32 c) {
-    const LanguageBreakEngine *lbe = NULL;
```

Comment:
this is surely wrong, LanguageBreakEngine should be immutable and not changed once created, you need to keep it as a const pointer.

Reply:
Fixed